src/main/java/com/icthh/xm/ms/entity/config/amazon/AmazonS3Template.java

```diff
@@ -73,7 +77,7 @@ public void save(String key, InputStream inputStream) throws IOException {
      * @param inputStream is the file that will be saved
      */
     @SneakyThrows
-    public String save(String bucket, String key, InputStream inputStream, Integer contentLength, String fileName) {
+    public UploadResultDto save(String bucket, String key, InputStream inputStream, Integer contentLength, String fileName) {
```

Comment:
AmazonS3Template in LEP context, and you must not change api.
Pls, add new method